Add information about platform and Pagar.me's module

leonampd opened the pull request:
### Description

Since we can't know what version of our magento module our users are
using, this implementation add a new header called `User-agent` to all
requests to Pagar.me's API. This header includes versions about Magento,
PHP and Pagar.me's magento module and will permit us to monitor wich
versions our clients are using

### Issue

There's no isues about this

### Tests

No tests created

henriquekano commented on app/code/community/Inovarti/Pagarme/Model/Api.php:
```diff
@@ -171,6 +171,21 @@ public function calculateInstallmentsAmount(Varien_Object $data)
 	}
 
+    /**
+     * Return a string with Magento, Pagarme SDK and PHP versions
```
Do you really need this commentary?

henriquekano commented on app/code/community/Inovarti/Pagarme/Model/Api.php:
```diff
@@ -171,6 +171,21 @@ public function calculateInstallmentsAmount(Varien_Object $data)
+     *
+     * @return string
+     */
+    public function getUserAgent()
```
The indentation in the entire file seem to be tab based

leonampd commented:
Hey @henriquekano can u review again, please? I converted all the file to use spaces and refactored some pieces of code to be ok with phpcs (code sniffer).
